perf/stability/mysql/setup.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
```

Comment:
The intent of the stability tests is to deploy common services (like mysql) and leave them running indefinitely. This seems more like an E2E test - we start up mysql, run a few queries and tear it down.

Reply:
See example: https://github.com/istio/tools/pull/95

In that PR I add some stuff to give us a smart client, so that we can report metrics from the client. Until that change is in though, you could just have the client make periodic read/writes?

Reply:
Interesting, thanks for the context.

- I do need to exercise MySQL connectivity under both mTLS and plain text. Should we create two deployments/svc, one enabled mtls one disabled?
- However, I personally feel we might not benefit a lot from long running MySQL under mTLS or plain text. If we screw up the tls related config, it fails very deterministically. But also it does no harm to test it though. :) 

@mandarjog  @howardjohn  Thoughts?

Reply:
That seems reasonable to me (to create 2 deployments)

Reply:
Done.

Two helm deployment using different name. CRD nested into helm as well.